tests/dev_wrappers/utils.py

```diff
@@ -0,0 +1,80 @@
+import typing
+from typing import Optional, Union
+
+import numpy as np
+
+import gymnasium
+from gymnasium import Space
+from gymnasium.core import ActType, ObsType
+from gymnasium.spaces import Box, Discrete
+
+
+class TestingEnv(gymnasium.Env):
```

Comment:
This already exists in `tests/testing_env.py`, could you modify that implementation instead of this

Reply:
https://github.com/Farama-Foundation/Gymnasium/pull/148/commits/846676a1cd84fffdb48b64c48db69b1dafac1843